### ISSUE TITLE
Bug: Syncing labels deletes *all* existing labels in a repo (and then repopulates)

### ISSUE THREAD

**Issue by shinytoyrobots** [carbon-design-system/sync#12]
This removes all the labels currently assigned to issues, which reduces the usefulness of the tool.

Desired functionality.

1. Tool should de-dupe and only add labels to a repo if those labels don't already exist
2. Tool should only delete labels that are not part of the master list (this should be optional)

**Comment by shinytoyrobots** [comment]
@joshblack - do you think this is an issue we can resolve? I don't really want to run sync on any of our major repos until this is fixed (because it would remove all the existing labels from all issues).

**Comment by joshblack** [comment]
@shinytoyrobots yeah, for sure. The script will need to be updated to check if the label exists already and if so don't clear it. I think right now it's a little naive where it will clear out everything potentially
